search/search_test.go

```diff
@@ -398,6 +400,34 @@ func TestTarjanSCC(t *testing.T) {
 	}
 }
 
+// batageljZaversnikGraph is the example graph from
```

Comment:
Or possibly not in internal if we want to expose some easy examples (like in optimize)